samples/iam_audit_log.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPIAMAuditLogConstraintV1
+metadata:
+  name: audit_test
```

Comment:
Sample constraint names should be descriptive (ie. not "testing.")

Reply:
updated the name